src/dune_file_watcher/dune_file_watcher.mli

```diff
@@ -76,5 +80,5 @@ val emit_sync : t -> Sync_id.t
 val add_watch : t -> Path.t -> (unit, [ `Does_not_exist ]) result
 
 module For_tests : sig
-  val should_exclude : string -> bool
+  val should_exclude : string -> string list -> bool
```

Comment:
Can you add a label to this argument? It's hard to understand the signature without it

Reply:
yep